.travis.yml

```diff
@@ -62,22 +60,22 @@ before_script:
 
 script:
   - >
-    if [[ $TRAVIS_PYTHON_VERSION_MINOR < 4 || $TRAVIS_PYTHON_VERSION_MAJOR == 2 ]]; then
+    if [[ "$UNSUPPORTED" == true ]]; then
       coverage run setup.py install | grep -q 'coala supports only python 3.4 or later'
-    else
-      set -e
-      bash .misc/tests.sh
-      python setup.py bdist_wheel
-      pip install ./dist/coala-*.whl
-      pip install coala-bears[alldeps] --pre -U
-      # https://github.com/coala/coala-bears/issues/1037
-      if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then
-        sed -i.bak '/bears = GitCommitBear/d' .coafile
-      fi
-      coala --non-interactive
-      make -C docs clean
-      python setup.py docs
     fi
+  - if [[ "$UNSUPPORTED" == true ]]; then codecov || true; exit; fi
+  - bash .misc/tests.sh
```

Comment:
Yes, travis does `set -e` by checking the error status of each entry in the build script.  It needs to be manually added only if there are multiple bash commands in a single entry of the build script, which travis cant 'see'.